pipelines/components/tensorflow/kmeansclustering/README.md

```diff
@@ -0,0 +1,24 @@
+# Component: KMeansClustering
```

Comment:
There is a doc (I will share its link with you) with some guideline about how to write the documentation. It might be a good idea to follow that to make sure all the README files have a similar structure.